nengo/linear_system.py

```diff
@@ -0,0 +1,369 @@
+import numpy as np
+
+from nengo.base import Process
+from nengo.exceptions import ValidationError
+from nengo.params import BoolParam, EnumParam, NdarrayParam, Parameter
+from nengo.rc import rc
+from nengo._vendor.scipy.signal import (
+    cont2discrete,
+    ss2tf,
+    tf2ss,
+    tf2zpk,
+    zpk2ss,
+)
+
+
+def _as2d(M, last=True, dtype=np.float64):
```

Comment:
Is there anywhere this is used with `last=True`? I only see the one use below (with `last=False`).

Reply:
No, it's not. I've changed it so that we use `last=True` on `C` and `D`, and `last=False` on `A` and `B`. Essentially this amounts to assuming the system is SISO if vectors are provided for `B` and `C`, which seems like a fine assumption to me (I doubt this will be used much).